apps/explorer/components/Contract/index.tsx

```diff
@@ -318,48 +334,49 @@ function isProperlyFormedRenewedContract(contract: SiaCentralContract) {
   } = getRenewedContractFormattedOutputs(contract)
 
   // renter payout valid and missed should be the same
-  if (renterPayoutValid.value !== renterPayoutMissed.value) {
+  if (renterPayoutValid?.value !== renterPayoutMissed?.value) {
     return false
   }
 
   // host payout valid and missed should be the same
-  if (hostPayoutValid.value !== hostPayoutMissed.value) {
+  if (hostPayoutValid?.value !== hostPayoutMissed?.value) {
     return false
   }
 
   // math.MaxUint64 with lost precision
   const mathMaxUint64 = 18446744073709552000
-  if (contract.revision_number !== mathMaxUint64) {
+  if (contract.fileContract.revisionNumber !== mathMaxUint64) {
     return false
   }
   return true
 }
 
-function getNewContractFormattedOutputs(contract: SiaCentralContract) {
+function getNewContractFormattedOutputs(contract: ExplorerFileContract) {
   return {
-    renterPayoutValid: contract.valid_proof_outputs[0],
-    renterPayoutMissed: contract.missed_proof_outputs[0],
-    hostPayoutValid: contract.valid_proof_outputs[1],
-    hostPayoutMissed: contract.missed_proof_outputs[1],
-    hostBurned: contract.missed_proof_outputs[2],
+    renterPayoutValid: contract.fileContract.validProofOutputs?.[0],
+    renterPayoutMissed: contract.fileContract.missedProofOutputs?.[0],
```

Comment:
We should double check with the team that all these assumptions around index position are still valid. But can also get a good idea from comparisons to production siascan.com.